src/Everwealth.OidcClient.Android/WebViewBrowser.cs

```diff
@@ -208,18 +220,16 @@ public override bool ShouldOverrideUrlLoading(WebView view, string url)
                     _webView.LoadUrl(_startUrl.ToString());
                     return true;
                 }
-                else if (uri.Host != _startUrl.Host)
+                else if (_viewableUrls.Any(x => Android.Net.Uri.Parse(x).Host == uri.Host))
```

Comment:
We'll need to add a null check here, as _viewableUrls may be null and calling an extension will throw an exception in this case.